### PR TITLE
Bibliography page

### PR DESCRIPTION
**Pull request by thewtex**
Start Citation page based on the 2019 Community Survey responses.

### CODE REVIEW

File: citations/ITKBibliography.bib [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+author = {Looney, P{\'{a}}draig and Stevenson, Gordon N. and Nicolaides, Kypros H. and Plasencia, Walter and Molloholli, Malid and Natsis, Stavros and Collins, Sally L.},
+doi = {10.1172/jci.insight.120178},
+file = {:Users/bpaniagua/Library/Application Support/Mendeley Desktop/Downloaded/Looney et al. - 2018 - Fully automated, real-time 3D ultrasound segmentation to estimate first trimester placental volume using deep lea.pdf:pdf},
```
[the remark this answers is not in the document]
Comment (bpaniagua) [reply citations/ITKBibliography.bib]: Yep, will correct!

File: citations/ITKBibliography.bib [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+archivePrefix = {arXiv},
+arxivId = {1811.05677},
+author = {Belmonte, Gina and Ciancia, Vincenzo and Latella, Diego and Massink, Mieke},
+eprint = {1811.05677},
+file = {:Users/bpaniagua/Library/Application Support/Mendeley Desktop/Downloaded/Belmonte et al. - 2018 - VoxLogicA a Spatial Model Checker for Declarative Image Analysis (Extended Version).pdf:pdf},
```
[the remark this answers is not in the document]
Comment (bpaniagua) [reply citations/ITKBibliography.bib]: Yep, will correct!

**Comment by bpaniagua** [comment]
@jhlegarreta please have a look!

File: citations/ITKBibliography.bib [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+month = {mar},
+number = {MAR},
+publisher = {Frontiers Research Foundation},
+title = {{Neuroinformatics and the the insight toolkit}},
+volume = {9},
+year = {2015}
+}
+@inproceedings{Yatabe2017,
+author = {Yatabe, Marilia and Ruellas, Antonio and Gomes, Liliane and Macron, Lucie and Lopinto, Julia and Paniagua, Beatriz and Budin, Francois and Cevidanes, Lucia},
+booktitle = {2017 IADR/AADR/CADR General Session (San Francisco, California) },
```
Comment (jhlegarreta) [inline citations/ITKBibliography.bib]: Unnecessary whitespace before right curly brace.

Comment (bpaniagua) [reply citations/ITKBibliography.bib]: What space? I cant see anything... :confused:

Comment (dzenanz) [reply citations/ITKBibliography.bib]: `) }` space between closing parenthesis and brace, at the end of the line

Comment (jhlegarreta) [reply citations/ITKBibliography.bib]: Exactly. Thanks @dzenanz . Sorry for not having been more explicit.

Comment (bpaniagua) [reply citations/ITKBibliography.bib]: wow, you guys have :eyes: 
Thanks for the clarification, fixed now (I think)

Comment (jhlegarreta) [reply citations/ITKBibliography.bib]: :+1: Thanks for having taken care of this @bpaniagua . Addressed in 9cee7c1.

**Comment by thewtex** [comment]
@jhlegarreta @bpaniagua thanks for the reviews and edits!

> Should these be included
https://itk.org/Wiki/ITK/FAQ#How_do_I_cite_the_use_of_ITK_in_a_publication.3F
?

I'll add pointer to this repository.
